must-gather/collection-scripts/gather_collection_resources

```diff
@@ -47,14 +47,20 @@ check_collector_persistence() {
 }
 
 echo "Gathering data for collection component"
-echo "-- Checking Collector health"
 mkdir -p $collector_folder
 
+echo "-- Retrieving configmaps"
+oc -n $NAMESPACE extract configmap/fluentd --to=$collector_folder ||:
+oc -n $NAMESPACE extract configmap/secure-forward --to=$collector_folder ||:
```

Comment:
Do we miss the syslog configmap here?

Reply:
Maybe.  I simply transfered lines from the tests into here and I did not see those